code/controllers/subsystem/persistence/trophy_fishes.dm

```diff
@@ -32,10 +32,9 @@
 		fish.set_custom_materials(mat_list)
 	fish.persistence_load(data)
 	fish.name = data[PERSISTENCE_FISH_NAME]
-	mount.catcher_name = data[PERSISTENCE_FISH_CATCHER]
```

Comment:
?

Reply:
It does, it's the same vale of the catcher arg, no?